### DISCUSSION TITLE
ScriptReference/Animations.PropertyStreamHandle.GetReadMask

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#551] [Notes]
# ScriptReference/Animations.PropertyStreamHandle.GetReadMask

https://docs.unity3d.com/ScriptReference/Animations.PropertyStreamHandle.GetReadMask.html

**Comment by WispBart** [comment]
Use this to see if the provided AnimationStream contains data for the PropertyStreamHandle.
